nautobot/dcim/tests/test_views.py

```diff
@@ -836,7 +836,7 @@ class ConsolePortTemplateTestCase(ViewTestCases.DeviceComponentTemplateViewTestC
 
     @classmethod
     def setUpTestData(cls):
-        manufacturer = Manufacturer.objects.create(name="Manufacturer 1", slug="manufacturer-1")
+        manufacturer = Manufacturer.objects.create(name="Manufacturer 1")
```

Comment:
Can't we use `Manufacturer.objects.first()` instead of creating a new Manufacturer here and throughout these tests?